apps/frontend/src/pages/landing/components/landing-cta/landing-cta.tsx

```diff
@@ -0,0 +1,27 @@
+import { Button, Link } from "~/libs/components/components.js";
+import { AppRoute, ButtonSize, ButtonVariant } from "~/libs/enums/enums.js";
+
+import styles from "./styles.module.css";
+
+const LandingCTA: React.FC = () => {
+	return (
+		<section aria-labelledby="cta" className={styles["cta"]}>
+			<div className={styles["cta__inner"]}>
+				<h2 className={styles["cta__title"]}>
```

Comment:
```suggestion 
<section aria-labelledby="cta-title" className={styles["cta"]}>
			<div className={styles["cta__inner"]}>
				<h2 id="cta-title" className={styles["cta__title"]}>
```
For `aria-labelledby` to work, the `<h2 />` heading must have an id attribute that matches the value of `aria-labelledby`.